articles/firewall-manager/migrate-to-policy.md

```diff
@@ -138,10 +153,10 @@ If ($azfw.NatRuleCollections.Count -gt 0) {
 		If ($rc.Rules.Count -gt 0) {
 			Write-Host "creating " $rc.Rules.Count " nat rules for collection "  $rc.Name
 			ForEach ($rule in $rc.Rules) {
-				$firewallPolicyNatRule = New-AzFirewallPolicyNetworkRule -Name $rule.Name -SourceAddress $rule.SourceAddresses -DestinationAddress $rule.DestinationAddresses -DestinationPort $rule.DestinationPorts -Protocol $rule.Protocols
+				$firewallPolicyNatRule = New-AzFirewallPolicyNatRule -Name $rule.Name -SourceAddress $rule.SourceAddresses -TranslatedAddress $rule.TranslatedAddress -TranslatedPort $rule.TranslatedPort -DestinationAddress $rule.DestinationAddresses -DestinationPort $rule.DestinationPorts -Protocol $rule.Protocols
 				Write-Host "Created nat rule " $firewallPolicyNatRule.Name
 				$natRuleCollectionName = $rc.Name+$rule.Name
-				$fwpNatRuleCollection = New-AzFirewallPolicyNatRuleCollection -Name $natRuleCollectionName -Priority $priority -ActionType $rc.Action.Type -Rule $firewallPolicyNatRule -TranslatedAddress $rule.TranslatedAddress -TranslatedPort $rule.TranslatedPort
+				$fwpNatRuleCollection = New-AzFirewallPolicyNatRuleCollection -Name $natRuleCollectionName -Priority $priority -ActionType $rc.Action.Type -Rule $firewallPolicyNatRule
```

Comment:
Thank you for updating the script. We not support multiple nat rules per collection. So we need not do a collection per rule.

Reply:
Great addition. Added support to that.